spec/dummy/config/webpacker.yml

```diff
@@ -56,7 +56,7 @@ development:
 
 test:
   <<: *default
-  compile: true
+  compile: false
```

Comment:
did you switch to the react on rails helper?

if so, we should encourage that in the docs